ADAL/src/cache/ADTokenCacheItem.m

```diff
@@ -121,7 +121,8 @@ - (BOOL)isSameUser:(ADTokenCacheItem*) other
     
     if ([self isEmptyUser])
         return [other isEmptyUser];
-    return (nil != other.userInformation && [_userInformation.userId isEqualToString:other.userInformation.userId]);
+    
+    return [self.userInformation isEqual:other.userInformation];
```

Comment:
sure, I can update the comment, but can we remove completely 
`- (BOOL)isSameUser:(ADTokenCacheItem *)other`
method? I've checked and it is not used in ADAL anymore, but it leaves in public headers, so it might affect other users of the library. @RPangrle @oldalton what do you guys think, can we remove it?